### DISCUSSION TITLE
Question wrt whdload_db.xml

### DISCUSSION THREAD

**Discussion by giantclambake** [BlitterStudio/amiberry#1112] [Q&A] [answered]
What does the XML  param SCREEN_AUTOHEIGHT=TRUE  map to in amiberry conf/display panel?

I think this is somehow overriding  SCREEN_CENTERH=SMART & SCREEN_CENTERV=SMART

TIA

**Comment by midwan** [comment, marked as answer]
Auto-Height is only used for backwards compatibility and for the DispmanX versions, which don't have AutoCrop (but has Auto-Height instead)

**Comment by giantclambake** [comment]
Thanks, understood  =)
